openhtf/output/proto/test_runs.proto

```diff
@@ -21,7 +21,7 @@ syntax = "proto2";
 package openhtf;
 
 // import "assembly.proto";
-import "units.proto";
+import "openhtf/output/proto/units.proto";
```

Comment:
This was an artifact of an older version of the proto runtime + compiler; the handling of relative imports changed. Because this proto includes its peer relative to the project root, I also altered the proto compiler invocation to run from the root of openhtf.